uber/templates/accounts/sitemap.html

```diff
@@ -6,10 +6,12 @@
 
 {% for section, pagelist in pages %}
     <div class="panel panel-default">
-    <div class="panel-heading">{{ section|title }}</div>
+    <div class="panel-heading"><a href="../{{ section }}/index">{{ section|title }}</a></div>
         <ul class="nav nav-pills">
         {% for page in pagelist %}
+            {% if page.name != 'Index' %}
```

Comment:
Eli is correct in that the index page is usually important. Even though you added it to the header (which is a good change!), people may miss having an explicit link.